examples/atari/train_acer_ale.py

```diff
@@ -4,8 +4,8 @@
 # Prevent numpy from using multiple threads
 os.environ["OMP_NUM_THREADS"] = "1"
 
-import gym  # NOQA:E402
-import gym.wrappers  # NOQA:E402
+import gymnasium  # NOQA:E402
```

Comment:
I'm not familiar with gymnasium much, but is it recommended to write `import gymnasium as gym`?
Do you know any article about coding convention of gymnasium?

Reply:
I'm not sure. The examples they give often write what you have, but I think it's just to sell the simplicity of their transition from gym to gymnasium. I also think the distinction can help for clarity, so people are reminded which API is being used.

Reply:
Hmm it does seem even internally in their code they use `gym` as you say.

Reply:
Thanks. Anyway I agree with your opinion:
> the distinction can help for clarity